src/Microsoft.Management.Infrastructure.CimCmdlets/project.json

```diff
@@ -19,6 +19,7 @@
             "buildOptions": {
                 "define": [ "CORECLR" ]
             },
+            "debugType": "portable"
```

Comment:
According to [project.json schema](http://json.schemastore.org/project), `debugType` is a property of the `buildOptions` section. So this should be included in the `buildOptions` section within the `netstandard1.6` section.